src/rtc-to-net/rtc-to-net.ts

```diff
@@ -175,7 +165,7 @@ module RtcToNet {
       });
       var writeSnapshot = () => {
         this.getSnapshot().then((snapshot:RtcToNetSnapshot) => {
-          log.info('snapshot: %1', [JSON.stringify(snapshot)]);
+          log.info('snapshot: ' + JSON.stringify(snapshot));
```

Comment:
Curious why you changed this -- the `%X` placeholders are intended to be faster, by avoiding the string concatenation.